### ISSUE TITLE
Vehicle set to speed 10 still moves at speed 4

### ISSUE THREAD

**Issue by bob-the-hamster** [ohrrpgce/ohrrpgce#1247]
Vehicles do not seem to be respecting speed settings as expected.

**Comment by rversteegen** [comment]
I tested in test.rpg, and found that the mounted vehicle speed is what's set as 'Speed' in the Vehicle editor, not what the NPC's speed is. I had completely forgotten that there was such a setting, and thought the NPC speed would be used. Maybe you did too?

**Comment by rversteegen** [comment]
I think this calls for a change to the NPC editor to indicate the vehicle's speed when mounted.

**Comment by bob-the-hamster** [comment]
No, I remember that vehicle speed is supposed to override NPC speed (but an indicator of that in the NPC editor is a great idea)

This comes from Wobbler. In Walthros Renewal, there is a vehicle with both NPC and vehicle speed set to 10 but it moves at 4.

This must be a more specific bug, not a general one. I'll go back and check what else could be happening. Perhaps a script manipulating hero speed?
